src/Microsoft.Framework.Caching.SqlServer/SqlServerCacheOptions.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Framework.Internal;
+using Microsoft.Framework.OptionsModel;
+
+namespace Microsoft.Framework.Caching.SqlServer
+{
+    public class SqlServerCacheOptions : IOptions<SqlServerCacheOptions>
```

Comment:
@kichalla and I agreed on remove the interface. Should we do it on the in-memory implementation as well?